.chloggen/prwreceiver-parselabels.yaml

```
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: receiver/prometheusremotewrite
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Parse labels from Prometheus Remote Write requests into Resource and Scope Attributes
```

```suggestion
note: Parse labels from Prometheus Remote Write requests into Resource and Metric Attributes.
```

Just noticed this now, I'll commit as soon as I see CI green